modules/ROOT/pages/_partials/supported-versions.adoc

```diff
@@ -46,7 +46,7 @@ Support for the following will be deprecated in this release and will be removed
 * macOS 11 - Big Sur
 * Apple OS X v10.14 (Mojave), v10.15 (Catalina)
 * Raspbian - 9
-* Debian 9
+* Debian 9, 10
```

Comment:
It's not the time to address this, but we should make sure that we are explicit about the releases here...is it 9.0 and 10.0? Or anything on either of these lines? Maybe a project to pursue when things settle down.